run_pacta_data_preparation.R

```diff
@@ -154,31 +153,60 @@ factset_timestamp <-
 
 # check that everything is ready to go -----------------------------------------
 
-stopifnot(file.exists(masterdata_ownership_path))
-stopifnot(file.exists(masterdata_debt_path))
-stopifnot(file.exists(ar_company_id__factset_entity_id_path))
-stopifnot(file.exists(factset_financial_data_path))
-stopifnot(file.exists(factset_entity_info_path))
-stopifnot(file.exists(factset_entity_financing_data_path))
-stopifnot(file.exists(factset_fund_data_path))
-stopifnot(file.exists(factset_isin_to_fund_table_path))
-stopifnot(file.exists(factset_iss_emissions_data_path))
-stopifnot(file.exists(factset_issue_code_bridge_path))
-stopifnot(file.exists(factset_industry_map_bridge_path))
-stopifnot(file.exists(factset_manual_pacta_sector_override_path))
-stopifnot(file.exists(data_prep_outputs_path))
+input_filepaths <- c(
+  masterdata_ownership_path = masterdata_ownership_path,
+  masterdata_debt_path = masterdata_debt_path,
+  ar_company_id__factset_entity_id_path = ar_company_id__factset_entity_id_path,
+  factset_financial_data_path = factset_financial_data_path,
+  factset_entity_info_path = factset_entity_info_path,
+  factset_entity_financing_data_path = factset_entity_financing_data_path,
+  factset_fund_data_path = factset_fund_data_path,
+  factset_isin_to_fund_table_path = factset_isin_to_fund_table_path,
+  factset_iss_emissions_data_path = factset_iss_emissions_data_path,
+  factset_issue_code_bridge_path = factset_issue_code_bridge_path,
+  factset_industry_map_bridge_path = factset_industry_map_bridge_path,
+  factset_manual_pacta_sector_override_path = factset_manual_pacta_sector_override_path
+)
 
 if (!update_currencies) {
-  stopifnot(file.exists(currencies_data_path))
+  input_filepaths <- c(
+    input_filepaths,
+    currencies_data_path = currencies_data_path
```

Comment:
```suggestion
    currencies_preflight_data_path = currencies_preflight_data_path
```
?

Reply:
e.g. if currencies are NOT going to be updated, then we're relying on a previously saved `currencies.rds` existing in the pre-flight path (which may or may not be the same as `data_prep_outputs_path`)